pkg/local_instance.go

```diff
@@ -206,16 +224,30 @@ func (li LocalInstance) Create() error {
 	if err := li.copyLicenseFile(); err != nil {
 		return err
 	}
-	if err := li.copyCbpExecutable(); err != nil {
+	if err := li.adapt(); err != nil {
 		return err
 	}
-	if err := li.correctFiles(); err != nil {
-		return err
+	log.Infof("%s > created", li.instance.ID())
+	return nil
+}
+
+func (li LocalInstance) Import() error {
+	log.Infof("%s > importing", li.instance.ID())
+
+	if !pathx.Exists(li.Dir()) {
+		return fmt.Errorf("%s >  %s dir doesn't exist", li.instance.ID(), li.Dir())
 	}
-	if err := li.createLock().Lock(); err != nil {
+
+	if !pathx.Exists(fmt.Sprintf("%s/crx-quickstart", li.Dir())) {
+		return fmt.Errorf("%s >  crx-quickstart folder is missing in unpack_dir: %s", li.instance.ID(), li.Dir())
```

Comment:
I have somewhere usages of `crx-quickstart` and I am referring to that thing just by saying `quickstart`

btw please use human language in messages; not code level details like `unpack_dir`

how about `quickstart dir does not exist at path '%s'`